Src/DependencyCollector/Shared/Implementation/HttpDiagnosticSourceListener.cs

```diff
@@ -47,14 +47,16 @@ public void OnNext(KeyValuePair<string, object> value)
             {
                 switch (value.Key)
                 {
-                    case "System.Net.Http.Request":
+                    case "System.Net.Http.Desktop.HttpRequestOut.Start": 
+                    case "System.Net.Http.Request": // remove in 2.5.0
```

Comment:
please expand the comment with the link to the issue or just write in words why it should be removed.